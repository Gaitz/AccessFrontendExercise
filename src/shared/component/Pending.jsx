import React from "react";
import styles from "./Pending.module.css";

const Pending = () => {
  return <p className={styles.pendingBlock}>Waiting for request </p>;
};

export default Pending;
