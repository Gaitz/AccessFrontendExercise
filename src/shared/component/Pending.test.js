import React from "react";
import Pending from "./Pending";
import { render } from "@testing-library/react";

describe("test Pending", () => {
  it("snapshot test", () => {
    const { container } = render(<Pending></Pending>);
    expect(container).toMatchSnapshot();
  });
});
