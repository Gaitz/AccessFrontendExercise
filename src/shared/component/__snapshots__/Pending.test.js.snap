// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Pending snapshot test 1`] = `
<div>
  <p
    class="pendingBlock"
  >
    Waiting for request
  </p>
</div>
`;
